package.json

```diff
@@ -83,6 +82,7 @@
     "performance-now": "^0.2.0",
     "shady-css-parser": "0.0.8",
     "split": "^1.0.0",
-    "strip-indent": "^2.0.0"
+    "strip-indent": "^2.0.0",
+    "typescript": "^2.1.4"
```

Comment:
They appear to be following semver... we can tighten over time. This isn't even fully wired in yet.